lib/bls_keystore.go

```diff
@@ -19,23 +19,26 @@ import (
 // We need to associate individual op-codes for each message type that can be signed, so that there is no risk
 // of signature collisions between different message types. The payload signed per message type must be made
 // up of the following tuples:
-// - Validator Vote:            (0x01, view uint64, blockHash consensus.BlockHash)
-// - Validator Timeout:         (0x02, view uint64, highQCView uint64)
-// - PoS Block Proposal:        (0x03, view uint64, blockHash consensus.BlockHash)
+// - PoS Validator Vote:        (0x01, view uint64, blockHash consensus.BlockHash)
+// - PoS Validator Timeout:     (0x02, view uint64, highQCView uint64)
+// - PoS Block Proposal:        (0x01, view uint64, blockHash consensus.BlockHash)
 // - PoS Validator Handshake:   (0x04, peer's random nonce, our node's random nonce)
 // - PoS Random Seed Signature: (previous block's random seed hash)
 
 type BLSSignatureOpCode byte
 
 const (
-	BLSSignatureOpCodeValidatorVote         BLSSignatureOpCode = 0
-	BLSSignatureOpCodeValidatorTimeout      BLSSignatureOpCode = 1
-	BLSSignatureOpCodePoSBlockProposal      BLSSignatureOpCode = 2
-	BLSSignatureOpCodePoSValidatorHandshake BLSSignatureOpCode = 3
+	BLSSignatureOpCodeValidatorVote         BLSSignatureOpCode = BLSSignatureOpCode(consensus.SignatureOpCodeValidatorVote)
+	BLSSignatureOpCodeValidatorTimeout      BLSSignatureOpCode = BLSSignatureOpCode(consensus.SignatureOpCodeValidatorTimeout)
+	BLSSignatureOpCodePoSValidatorHandshake BLSSignatureOpCode = 4
```

Comment:
Should this be 3 instead of 4 if we're getting rid of the OpCodePosBlockProposal?